src/openrct2/management/Research.cpp

```diff
@@ -611,24 +724,30 @@ void research_set_priority(sint32 activeCategories)
  *
  *  rct2: 0x00684A7F
  */
-void game_command_set_research_funding(sint32* eax, sint32* ebx, sint32* ecx, sint32* edx, sint32* esi, sint32* edi, sint32* ebp)
+void
+game_command_set_research_funding(sint32 * eax, sint32 * ebx, sint32 * ecx, sint32 * edx, sint32 * esi, sint32 * edi, sint32 * ebp)
```

Comment:
that's a bit odd formatting here, the preferred form is to wrap arguments

Reply:
Autoformatter :/